### PR TITLE
#985 Independent assessor change

### PR DESCRIPTION
**Pull request by lloback**
Change assessor

Changes the `application` and `assessor` if exists

### CODE REVIEW

File: src/views/Exercises/Applications/Application.vue [hunk 1]
```diff
@@ -2175,6 +2177,34 @@ export default {
         this.$store.dispatch('application/update', { data: { [field]: val }, id: this.applicationId });
       }
     },
+    editAssessor(AssessorNr) {
+      if (AssessorNr === 1) {
+        this.assessorDetails = {
+          AssessorNr: AssessorNr,
+          applicationId: this.application.id,
+          firstAssessorEmail: this.application.firstAssessorEmail,
```
Comment (warrensearle) [inline src/views/Exercises/Applications/Application.vue:2185]: I think you could pass more generic data/props to the modal like `email`, `name` etc along with `assessorNumber` which should mean the modal form and the store can be simplified.

**Review by warrensearle — COMMENTED** [review]
Please see comment. Whilst this is here in a PR I think we should simplify

File: src/views/Exercises/Applications/Application.vue (hunk 1 above)
Comment (lloback) [reply src/views/Exercises/Applications/Application.vue]: This has been implemented - a cleaner form

**Review by warrensearle — APPROVED** [review]
Looks good to me 👍

**Review by mbrookeswebdev — APPROVED** [review]

**Pull request merged**